PathPlanning/Bug_Planning/bug.py

```diff
@@ -0,0 +1,296 @@
+"""
+Bug Planning
+author: Sarim Mehdi(muhammadsarim.mehdi@studio.unibo.it)
+Source: https://sites.google.com/site/ece452bugalgorithms/
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+class Bug():
+    def __init__(self, startx, starty, goalx, goaly, obsx, obsy):
```

Comment:
variable name convention: could you please all variable should use `_` between words. For example,
startx -> start_x
obsx -> obs_x
Could you please fix all variable names?